### DISCUSSION TITLE
Only one usable ADC pin exposed?

### DISCUSSION THREAD

**Discussion by peteDDD** [witnessmenow/ESP32-Cheap-Yellow-Display#124] [General]
I've built a few useful things now with the CYD and my SD breakout card for digital IO.  However, my latest project for which I was hoping to use this board requires four ADC channels.   Since you can't use ADC2 when using WiFi, it appears that the only exposed, usable, ADC pin is IO35 ADC1_CH37 on the P3 connector.  All other available pins, even counting the three used for the RGB LED do not present any other of the ADC1 pins.  Does anyone disagree?

Very short-sighted design to not leave ADC_1 pins exposed and use ADC_2 pins for other on-board digital IO instead of wasting the ADC_1 pins.  If this board were to be redesigned, IO33/ADC1_CH5 and IO32/ADC1_CH7 which are currently used for CS and Din on the touch chip, could be swapped for IO16 and IO17 currently used by the RGB LED to bring the potential exposed ADC1 pins up to 3.  Also, IO34 ADC1_CH6 appears to be unused and should be exposed providing a total of 4 exposed ADC1 pins (which can also be used for GPIO).  What a waste...

**Comment by Skilledcamman** [comment]
Potentially add another micro controller using serial.
